### DISCUSSION TITLE
Renovate picking up an old image tag - linuxserver/heimdall

### DISCUSSION THREAD

**Discussion by chrede88** [renovatebot/renovate#25592] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Renovate is picking up an old image tag for heimdall (https://hub.docker.com/r/linuxserver/heimdall). My manifest is set to version `2.5.7` which is the latest stable version, however Renovate wants to "update" to the `2021.11.28` tag. This tag is over 2 years old. I guess the issue is more the very inconsistent use of tags by the linuxserver people, but is there anything I can do to prevent this?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by viceice** [comment, marked as answer]
use allowedVersions config option
